conf/data-transformers/Camunda.data-transformers.yml

```diff
@@ -0,0 +1,10 @@
+settings: 
+    - src_table: "ACT_HI_PROCINST"
+      dest_table: "process"
+      column_mapping: 
+        DURATION_: "duration"
+        PROC_INST_ID_: "process_instance_id"
+        START_TIME_: "start_time"
+        END_TIME_: "end_time"
+      column_transformation: 
+        DURATION_: "secondsToMilliseconds"
```

Comment:
@VincenzoFerme My mistake. I wanted to keep it just to remember it's there if needed.